### PR TITLE
[calibre] new chart

### PR DESCRIPTION
**Pull request by rolfberkenbosch**
**Description of the change**
Calibre chart with webbased acces to the desktop app. Very handy!

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [x] (optional) Chart is using our [common](https://github.com/k8s-at-home/charts/tree/master/charts/common) library as a dependency.
- [x] (optional) Variables are documented in the README.md

### CODE REVIEW

File: charts/calibre/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+
+apiVersion: v2
+appVersion: 1.0.0
```
Comment (onedr0p) [inline charts/calibre/Chart.yaml:4]: This needs to be 5.13.0

File: charts/calibre/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+description: Calibre is a powerful and easy to use e-book manager.
+name: calibre
+version: 5.13.0
```
Comment (onedr0p) [inline charts/calibre/Chart.yaml:7]: This needs to be 1.0.0

**Comment by onedr0p** [comment]
Just noticed this the version mistake on wikijs too, I opened a PR for that here https://github.com/k8s-at-home/charts/pull/666

I had to bump the version to 3.0.0 because we cannot "downgrade" a version.

File: charts/calibre/Chart.yaml (hunk 1 above)
Comment (rolfberkenbosch) [reply charts/calibre/Chart.yaml]: Ik fix the problem.

File: charts/calibre/Chart.yaml (hunk 2 above)
Comment (rolfberkenbosch) [reply charts/calibre/Chart.yaml]: Ik fix the problem.

**Review by onedr0p — APPROVED** [review]

**Comment by onedr0p** [comment]
@all-contributors please add @rolfberkenbosch for code

**Pull request merged**